client-bagel-buddy/src/components/card/index.css

```diff
@@ -1,16 +1,38 @@
 .card {
     border: 1px solid var(--card-border-color);
+    border-radius: 15px;
     padding: 16px;
     width: calc(25% - 50px);
+    fill: white;
+    box-shadow: 2px 2px 3px var(--card-border-color);;
 }
 
 .card-img {
     height: 225px;
     width: 100%;
+    border-radius: 10px;
 }
 
 .skill-list {
     display: flex;
     flex-wrap: wrap;
     gap: 10px;
-}
\ No newline at end of file
+    margin-top: 10px;
+}
+
+.skill-chip {
+    border: 1px solid #fff0e1;
```

Comment:
Let's put `#fff0e1` to `index.css` 's `:root` to set it as a global variable. In this way, if the design decides to change the color, we only need to change it in the `index.css` file :D